tendrl/commons/objects/cluster/atoms/delete_cluster_details/__init__.py

```diff
@@ -83,6 +84,32 @@ def run(self):
             # No cluster alerts, continue
             pass
 
+        # Remove cluster indexes for down node
+        try:
+            index_key = "/indexes/tags/tendrl/integration/%s" % integration_id
+            _node_ids = etcd_utils.read(
+                index_key
+            ).value
+            _node_ids = json.loads(_node_ids)
+            for _node_id in _node_ids[:]:
```

Comment:
Why `_node_ids[:]` ?

Reply:
it will consider list length every time, in the same list if any node is down I am removing the entry. so it won't give IndexError.

Reply:
Ack